tests/integration/actions/test_draft_datasource_config_action.py

```diff
@@ -66,6 +66,7 @@ def test_running_draft_datasource_config_action(
         "system_users",
         "checkpoints",
         "organizations",
+        "organizations_auth0_orgs",
```

Comment:
Any idea why this was added?

Reply:
sorry I should have added an explanation here: 

1. Lakitu created a auth0 table in mercury as part of their work here: https://github.com/greatexpectationslabs/mercury/pull/4141, which is being now being added to the DB that is used for integration tests for `cloud` and `runner`.
2. Runner has the equivalent change in the corresponding test [here](https://github.com/greatexpectationslabs/gx-runner/blob/3696e3b939b4f194fe887a173347d7e475c29de9/tests/integration/actions/test_draft_datasource_config_action.py#L68)

Reply:
All good - thanks for explaining here!